src/components/Card/Card.js

```diff
@@ -42,4 +43,8 @@ function Card( { pokemon: { name, id, sprites, types }, onClick } ) {
 	);
 }
 
+function useClassName( { types } ) {
```

Comment:
Why a custom hook? seems like just a utility function? (or at least, make this custom hook reusable and open for prefix/suffix + return an object if doing so).

Reply:
Moved to component